Formula/fn.rb

```diff
@@ -45,6 +45,7 @@ def install
       ENV["FN_REGISTRY"] = "fnproject"
       expected = "Successfully created app:  myapp"
       output = shell_output("#{bin}/fn create app myapp")
+      sleep 1
```

Comment:
Why would it need to sleep before validating a string?

Reply:
That was in the wrong place, at the very least 🤦🏻‍♂️.

Reply:
> Why would it need to sleep before validating a string?

To be fresh and energised for the tedious work of string validation